src/tests/test_rng.cpp

```diff
@@ -15,6 +16,16 @@
   #include <botan/x931_rng.h>
 #endif
 
+#if defined(BOTAN_HAS_ENTROPY_SOURCE)
+  #include <botan/entropy_src.h>
+#endif
+
+#if defined(BOTAN_TARGET_OS_TYPE_IS_UNIX)
+  #include <unistd.h>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the whitespaces before the # character. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=cpp%3APPDirectiveIndentation)